README.md

```diff
@@ -52,13 +52,12 @@ jobs:
 ```
 
 ## Run MATLAB Command
-When you define your workflow in the `.github/workflows` directory of your repository, specify the **Run MATLAB Command** action as `matlab-actions/run-command@v1`. The action requires an input.
+When you define your workflow in the `.github/workflows` directory of your repository, specify the **Run MATLAB Command** action as `matlab-actions/run-command@v1`. The action requires an input and accepts an optional input.
```

Comment:
just a thought: feel free to ignore it:

while very concise and crisp, at first read it felt as though the action requires an input _but_ can tolerate _it_ being optionally specified although that thought doesn't make much sense. I guess it is probably because the `input` referred to as required vs optional seemed indistinguishable, personally at first glance.

at the risk of an additional word, `The action requires an input and accepts an additional optional input`? Alternatively, is this sentence required given the table below starts the `Description `with `Required `or `Optional`

Reply:
Hi, Ajay. Good point. Yes, I am not sure about keeping this sentence either. I am going to send the text to Erica, our editor, and ask her what to do. It might be a good idea to remove the sentence altogether.

Reply:
After discussing with Erica, we decided to keep the sentence, but we added the word "also" for clarity.